.github/workflows/count-git-contributions.yml

```
@@ -0,0 +1,136 @@
+name: cronjob for counting git contributions and filing a PR with results
+
+# Run every Monday at 8:00 UTC
+# On Monday, we count git contributions
+# But we can also manually trigger this
+on:
+  schedule:
+    - cron: "0 14 * * MON"
+  workflow_dispatch:
+
```

Since you're using `peter-evans/create-pull-request` later on, you'll need to allow the actions context permission to open a pull request.

I think the documentation on the action is a little outdated here, but this is how they say to do it:
https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#workflow-permissions

```suggestion
permissions:
  pull-requests: write
```